src/utils/csv.ts

```diff
@@ -0,0 +1,146 @@
+import { Aggregate, DatapointsGetAggregateDatapoint } from '@cognite/sdk';
+import moment from 'moment-timezone';
+import { getGranularityInSeconds } from './utils';
+
+export interface DatapointsToCSVProps {
+  data: DatapointsGetAggregateDatapoint[];
+  granularity: string;
+  aggregate: Aggregate;
+  delimiter: Delimiters;
+  format?: string;
+}
+
+export enum Delimiters {
+  Coma = ',',
+  Tab = 'tab',
+  Semicolon = ';',
+  TabSign = '\t',
```

Comment:
very confusing type actually
I was thinking about
```
enum {
    name: sign
}
```
And then you can use both name (for visual) and sign (for logic) in your code.
Like here
https://spreadsheeto.com/wp-content/uploads/2017/10/comma-delimiter.png
Idk the reason we display `;` and `,` as signs and `tab` as name instead

Reply:
the code above can be just generated from the correct enum instead